index.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style/style.css">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Apple Page</title>
+</head>
+<body>
+    <header>
+        <nav class="navbar">
+            <div class="navbar-logo">
+                <a href="#" class="logo"><img src="images/apple-xxl.png" alt="Apple Store"></a>
+            </div>
+            <div class="navbar-elements">
+                <a href="#" class="elements">Store</a>
+                <a href="#" class="elements">Mac</a>
+                <a href="#" class="elements">iPod</a>
+                <a href="#" class="elements">iPhone</a>
+                <a href="#" class="elements">iPad</a>
+                <a href="#" class="elements">iTunes</a>
+                <a href="#" class="elements">Support</a>
+            </div>
+            <div class="navbar-search">
+                <form class="search">
+                    <div class="input">
+                        <i class="fa fa-search"></i>
+                    </div>
+                    <input type="text" class="input-search">
+                </form>
+            </div>
+        </nav>
+    </header>
+    <main class="container">
+        <div class="centerdiv">
+            <div class="center-container">
+                <div class="center">
+                    <div class="background"></div>
```

Comment:
**Optional consideration:** Is it not possible to have these classes set on only one div and achieve the same result? :thinking: